### PR TITLE
All triggers and unittests

### PR DESCRIPTION
**Pull request by Ashley-Bass**

### CODE REVIEW

File: src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                AccountId = message.AccountId,
+                PeriodYear = ConvertDateToPeriodYear(DateTime.Today),
+                PeriodMonth = (short)DateTime.Today.Month
+            };
+            await handler.Handle(convertedMessage);
+        }
+
+        public static string ConvertDateToPeriodYear(DateTime date)
+        {
+            return date.Year.ToString().Substring(2, 2) + "/" +
```
Comment (cofaulco) [inline src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs]: PeriodYear should be "19-20" format

File: src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            var convertedMessage = new RefreshEmployerLevyDataCompletedEvent
+            {
+                AccountId = message.AccountId,
+                PeriodYear = ConvertDateToPeriodYear(DateTime.Today),
+                PeriodMonth = (short)DateTime.Today.Month
+            };
+            await handler.Handle(convertedMessage);
+        }
+
+        public static string ConvertDateToPeriodYear(DateTime date)
```
Comment (cofaulco) [inline src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs]: This should be a private method

File: src/SFA.DAS.Forecasting.Functions.Triggers.UnitTests/WhenAccountFundsExpiredTriggered.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        [Test]
+        [Category("UnitTest")]
+        [TestCase("01/01/2019","19/20")]
+        [TestCase("01/01/2000","00/01")]
+        [TestCase("01/01/2099","99/00")]
+        public void Then_Period_Year_Is_Properly_Created(DateTime date, string expectedPeriodYear)
+        {
+            //Act
+            var res = HandleAccountFundsExpiredEvent.ConvertDateToPeriodYear(date);
```
Comment (cofaulco) [inline src/SFA.DAS.Forecasting.Functions.Triggers.UnitTests/WhenAccountFundsExpiredTriggered.cs]: should be testing the public interface. The Convert method should be private

File: src/SFA.DAS.Forecasting.Jobs.Application/Triggers/Handlers/PaymentCompleteTriggerHandler.cs [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                }
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex, $"Failed to trigger Payment PreLoad HttpTriggerFunction for AccountId: {refreshPaymentDataCompletedEvent.AccountId}, PeriodEnd: {refreshPaymentDataCompletedEvent.PeriodEnd}");
+                throw;
+            }
+        }
+
+        public static PeriodDate GetPeriodDateFromPeriodId(string periodId)
```
Comment (cofaulco) [inline src/SFA.DAS.Forecasting.Jobs.Application/Triggers/Handlers/PaymentCompleteTriggerHandler.cs]: This method should be private

File: src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs (hunk 2 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs]: Been made private

File: src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs (hunk 1 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.Forecasting.Triggers/HandleAccountFundsExpiredEvent.cs]: Corrected

File: src/SFA.DAS.Forecasting.Functions.Triggers.UnitTests/WhenAccountFundsExpiredTriggered.cs (hunk 3 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.Forecasting.Functions.Triggers.UnitTests/WhenAccountFundsExpiredTriggered.cs]: Sorted

File: src/SFA.DAS.Forecasting.Jobs.Application/Triggers/Handlers/PaymentCompleteTriggerHandler.cs (hunk 4 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.Forecasting.Jobs.Application/Triggers/Handlers/PaymentCompleteTriggerHandler.cs]: Sorted

**Pull request merged**